### PR TITLE
Feature/Add support for multiple regions

### PR DESCRIPTION
**Pull request by activehigh**
We have updated the plugin on our side to create ECR in multiple regions to better increase resilience in case one of the regions goes down. 

Just wanted to add this feature to the plugin so that others looking for something similar can take the benefit.

### CODE REVIEW

File: hooks/pre-command [hunk 1]
```diff
@@ -42,40 +47,69 @@ upsert_ecr() {
...
+    upsert_ecr "${repository_name}" "${region}"
+  done
+}
+
+get_regions() {
+  regions=()
+
+  if [ -z "${BUILDKITE_PLUGIN_CREATE_ECR_REGIONS_0:-}" ]; then
+    echo "--- No regions mentioned. Getting current region"
+    fallback_region=$(aws ec2 describe-availability-zones --output text --query 'AvailabilityZones[0].[RegionName]')
```
Comment (72636c) [inline hooks/pre-command]: Is it possible for a build agent to be running unprivileged without `ec2:DescribeAvailabilityZones`? I wonder if something like `"${AWS_DEFAULT_REGION:-$(aws configure get region)}"` would suffice as a backward-compatible alternative.

File: README.md [hunk 2]
```diff
@@ -43,6 +43,21 @@ steps:
       - seek-oss/create-ecr#v1.3.0:
           name: my-repo
           repository-tags-file: path/to/repository-tags-file.json
+
```
Comment (72636c) [inline README.md]:
```suggestion
```

**Review by 72636c — COMMENTED** [review]
This is fantastic, thanks for the contribution! Apologies that it took a while to get to.

@Activehigh could you enable edits from maintainers?

File: hooks/pre-command (hunk 1 above)
Comment (activehigh) [reply hooks/pre-command]: Yeah, I guess this would be better. Thanks for the suggestion. Sending in an update.

**Comment by activehigh** [comment]
@72636c Don't see an option to enable edits from maintainers. Probably need to recreate the PR to do that, do you want me to create a new PR?

**Comment by 72636c** [comment]
@Activehigh ah that's weird. Feel free to try a new PR.

**Comment by activehigh** [comment]
Will create a separate PR

**Pull request closed without merging**